api/src/controllers/TodoController/index.ts

```diff
@@ -80,16 +81,10 @@ const get = async (req: Request, res: Response) => {
 const post = async (req: Request, res: Response) => {
   const { title, completed = false } = req.body
   try {
-    pool.query(
-      'INSERT INTO todos (title, completed) VALUES (?, ?)',
-      [title, completed],
-      (err) => {
-        if (err) return logger.log(err)
-        res.redirect(ROUTES.todos)
-      },
-    )
+    await db.insert(todos).values([{ title, completed }])
```

Comment:
Add validation

Reply:
research by validation on app/framework level
in php we have something like this, it calls JsonRequestValidator, and is used in annotations
<img width="1083" alt="image" src="https://github.com/midstem/space-h/assets/39136103/8a009e9b-f74a-410b-b00d-321fb055f2b2">

App automatically use this validator during request